scio-google-cloud-platform/src/main/java/com/spotify/scio/bigtable/BigtableBulkWriter.java

```diff
@@ -142,17 +142,13 @@ public void processElement(@Element Iterable<KV<ByteString, Iterable<Mutation>>>
 
     @FinishBundle
     public void finishBundle() throws Exception {
-      bigtableWriter.flush();
-      checkForFailures(failures);
-      LOG.debug("Wrote {} records", recordsWritten);
-    }
-
-    @Teardown
-    public void tearDown() throws Exception {
```

Comment:
Isn't there an overhead when creating a new bigbableWriter on each bundle?

Reply:
probably yes, but I don't know how to workaround this since `close` is the only way to flush now.
I'll simplify the code to remove the null check in the `startBundle` as we'll always have to create a new writer

Reply:
checked in the beam implementation, they also recreate writer on every bundle